samples/openapi3/client/petstore/typescript/builds/default/util.ts

```diff
@@ -26,3 +26,17 @@ export function isCodeInRange(codeRange: string, code: number): boolean {
         return true;
     }
 }
+
+/**
+* Returns if it can consume form
+*
+* @param consumes array
+*/
+export function canConsumeForm(contentTypes: string[]): boolean {
+    for (const contentType of contentTypes) {
+        if ('multipart/form-data' === contentType) {
+            return true;
+        }
+    }
+    return false;
```

Comment:
is it the same as just `return contentTypes.includes('multipart/form-data')`?

Reply:
I try to use includes and I got an error 
`Property ‘includes’ does not exist on type ‘string[]’. Do you need to change your target library? Try changing the ‘lib’ compiler option to ‘es2016’ or later.ts`
it breaks some apps I think  
lets use the old version or `arr.indexOf(valueToCheck) !== -1`
What do you think?
CC @amakhrov @macjohnny

Reply:
`indexOf` sounds good.